dist/source/wrapper/DdUrlTransfer.brs

```diff
@@ -579,11 +570,31 @@ function __DdUrlTransfer_builder()
     instance.ClearCookies = sub()
         m.roUrlTransfer.ClearCookies()
     end sub
+    instance._traceRequest = sub()
+        m.traceId = generateUniqueId()
+        m.spanId = generateUniqueId()
+        rndTrace = (Rnd(101) - 1) ' Rnd(n) returns a number between 1 and n (both inclusive)
+        ddLogWarning("rndTrace:" + rndTrace.toStr())
+        ddLogWarning("m.tracingSamplingRate:" + m.tracingSamplingRate.toStr())
+        if (rndTrace < m.tracingSamplingRate)
```

Comment:
We currently don't have any firstPartyHost mechanism to help decide which requests to trace in Roku?